<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACameraControlDialog</class>
 <widget class="QDialog" name="ACameraControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Lower-left</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Upper-right</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ledRangeLX">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="ledRangeUX">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ledRangeLY">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="ledRangeUY">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="ledRangeLZ">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="ledRangeUZ">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="5">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Rotation
center</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ledCenterX">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ledCenterY">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="ledCenterZ">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Center</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ledWx">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ledWy">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>Viewport</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="ledWw">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="ledWh">
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="1">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>Latitude</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>Longitude</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>Psi</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ledLat">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>º   </string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="ledLong">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>º   </string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="ledPsi">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_39">
       <property name="text">
        <string>º</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbDummy">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSetView">
       <property name="text">
        <string>...SetView</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ledRangeLX</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledRangeLY</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledRangeLZ</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledRangeUX</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledRangeUY</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledRangeUZ</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledWx</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledWy</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledWw</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledWh</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledLat</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledLong</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>298</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledPsi</sender>
   <signal>editingFinished()</signal>
   <receiver>pbSetView</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
